spec/features/reactivating_a_user_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+RSpec.feature 'Reactivating a user' do
+  let(:user) { FactoryBot.create(:user, auth_id: nil) }
+
+  before do
+    stub_auth0_token_request
+    sign_in_as_admin
+  end
+
+  scenario 'successfully' do
+    stub_auth0_create_user_request(user.email)
+
+    click_on 'Users'
+    click_on user.name
+    click_on 'Reactivate user'
+    click_on 'Reactivate user'
+
+    expect(page).to_not have_content('Inactive user')
+  end
+
+  scenario 'unsuccessfully with Auth0 error' do
+    stub_auth0_create_user_request_failure(user.email)
+
+    click_on 'Users'
+    click_on user.name
+    click_on 'Reactivate user'
+    click_on 'Reactivate user'
```

Comment:
There are two `'Reactivate user'` clicks here and in the `scenario` above. Is that intentional?

Reply:
Yes, because you have the `Reactivate user` button on the user page, and then the `Reactivate user` button on the interstitial page! It's the same as https://github.com/dxw/DataSubmissionServiceAPI/blob/develop/spec/features/deleting_a_user_spec.rb#L15

Reply:
👍 thanks